src/dragonball/src/device_manager/mod.rs

```diff
@@ -29,6 +29,12 @@ use dbs_virtio_devices::{
     VirtioDevice,
 };
 
+#[cfg(feature = "hotplug")]
```

Comment:
Would love to see an abstraction somewhere to allow `ACPI` hotplug implementation to happen. It can be done in later PRs of course.